app/views/staff/organisations/_grouped_programmes.html.haml

```diff
@@ -25,4 +25,6 @@
                   %td{class: "govuk-table__cell"}
                     = activity.roda_identifier_compound
                   %td{class: "govuk-table__cell"}
-                    = link_to(t("default.link.view"), organisation_activity_details_path(activity.extending_organisation, activity), class: "govuk-link")
+                    - if policy(:project).create?
+                      = link_to(t("action.activity.add_child"), organisation_activity_children_path(activity.extending_organisation, activity), method: :post, class: "govuk-link govuk-!-margin-right-4")
```

Comment:
Just wondering if this should be a `button_to` as it's an action that results in an activity being created? The `view` link could then be linked to the activity title

Reply:
I did wonder about that - there are several places where we have "View" links and others where we make the title clickable. The only problem with the latter is how you render empty titles.

I'm never sure about having rows of "primary action" style buttons on the same page. Might see if I the design system will let me style a button *as* a link.

Reply:
Think I'm going to leave this alone for now, even though it's not ideal.